charts/trino/values.yaml

```
@@ -948,3 +948,46 @@ ingress:
   #    hosts:
   #      - chart-example.local
   # ```
+
+networkPolicy:
+  enabled: false
+  # networkPolicy.enabled -- Set to true to enable Trino Pod protection with a [Network
+  # Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+  # By default, the policy will only allow Trino Pods to communicate with each other.
```

I also wonder if we need another note saying that when using JMX exporter and/or Service Monitor, additional rules for Prometheus might be required.
```suggestion
  # networkPolicy.enabled -- Set to true to enable Trino pod protection with a [Network
  # Policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
  # By default, the policy will only allow Trino pods to communicate with each other.
```

---

Makes sense, I will add a note in the documentation.